src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/ChangesInPXCache/ChangesInPXCacheDuringPXGraphInitializationTests.cs

```diff
@@ -11,7 +12,7 @@ namespace Acuminator.Tests.Tests.StaticAnalysis.ChangesInPXCache
     public class ChangesInPXCacheDuringPXGraphInitializationTests : DiagnosticVerifier
     {
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() =>
-            new PXGraphAnalyzer(new ChangesInPXCacheDuringPXGraphInitializationAnalyzer());
+            new PXGraphAnalyzer(CodeAnalysisSettings.Default, new ChangesInPXCacheDuringPXGraphInitializationAnalyzer());
```

Comment:
Don't you need `WithRecursiveAnalysisEnabled()` here? Tests should not rely on the default settings because they can change in the future; all settings that are used in the tests must be set explicitly.

Reply:
Ok